GA_analysis.Rmd

```diff
@@ -32,12 +32,18 @@ theme_set(theme_linedraw() +
           )
 ```
 
+```{r echo = FALSE, message = FALSE}
+# Need to authorize before this can be run. 
+ga_auth(email = "cansav09@gmail.com")
+ggmap::register_google(key = "AIzaSyBahIBC_aTnJjCwey8oEA1ngvyKo4_cZ60")
```

Comment:
Accidentally pushed this key so I've already delete the project it goes to so no one can get in there.